src/org/labkey/targetedms/parser/skyd/ChromGroupHeaderInfo.java

```diff
@@ -247,9 +259,6 @@ public boolean isNegativePolarity() {
     }
 
     public boolean excludesTime(double time) {
-        if (null == startTime || null == endTime) {
-            return false;
-        }
         return startTime > time || endTime < time;
```

Comment:
This needs to treat the placeholder values as if they were null and return false.